tests/python/relay/test_pass_simplify_expr.py

```diff
@@ -106,10 +106,28 @@ def expected3():
         y = relay.transpose(y, axes=[0, 2, 3, 1])
         return relay.Function([x], y)
 
+    # Test a series of transpose and rank changing layout_transform
```

Comment:
This test case looks great. Pretty clear to understand what is happening.
I think it would be great we can add one more fusion scenario as well, like some of the test cases before and the new one.
Please let me know in case i am not clear.

Reply:
I added many more variants :), please review them

Reply:
Thanks. Looks great now!